src/DssSpell.t.sol

```diff
@@ -666,6 +654,19 @@ contract DssSpellTest is DssSpellTestBase {
         }
     }
 
+    function testMKRPaymentsSum() public { // make public to enable
+        uint256 sumPayments = 6.34 ether; // NOTE: ether is a keyword helper, only MKR is transferred here
```

Comment:
Same here, let's move it to `testMKRPayments` with a nice comment suggesting to fill this number in from the executive doc